src/stuf/stuf_bg/constants.py

```diff
@@ -17,6 +24,37 @@
 STUF_BG_EXPIRY_MINUTES = 5
 
 
+type DateOfBirthPrecisionType = Literal["date", "year_month", "year"]
+
+
+class NaturalPersonDetails(BaseModel):
+    bsn: str | None = None
```

Comment:
why/how can these string fields be `None`? Shouldn't those be normalized to empty strings?